server/routes/index.js

```diff
@@ -9,6 +9,7 @@ r.get('/', index);
 r.get('/healthcheck', healthcheck);
 r.get('/explore', explore);
 r.get('/articles', articles);
+r.get('/articles/(W):id', prismicArticle);
```

Comment:
Does this mean 'the-`id`-is-whatever-this-string-is-without-a-`W`-at-the-front-if-there-is-one'?